src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -224,17 +237,94 @@ and one pod for any other component that you have enabled.
+
+To configure the individual integration components, you must use Helm's dependency system, which simply means that configuration for the children charts must be put under a section with the name of said chart. For example, to configure the `newrelic-infrastructure` chart, you would add the following to the `values-newrelic.yaml`:
+
+```yaml
+global:
+  licenseKey: <var>YOUR_NEW_RELIC_LICENSE_KEY</var>
+  cluster: <var>K8S_CLUSTER_NAME</var>
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  verboseLog: true  # Enable debug logs
+  privileged: false  # Install with minimal privileges
+  # Other options from https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure-v3
+```
+
+The full list of flags that can be tweaked can be found in our chart's repository:
+* [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure-v3)
+ * Configure debug logs, privilege mode, control plane monitoring, etc.
+* [`nri-kube-events`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-kube-events)
+* [`nri-metadata-injection`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-metadata-injection)
+ * Configure how the webhook for APM linkage is deployed.
+* [`nri-prometheus`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-prometheus)
+ * Configure which prometheus endpoints are scraped.
+* [`newrelic-logging`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging)
+ * Configure which logs are sent to New Relic.
+
+<Callout variant="tip">
+  Remember that when specifying options for these charts, you must put them under the chart name in your `values-newrelic.yaml`.
+</Callout>
+
+### Change the scrape interval [#scrape-interval]
+
+The Kubernetes Integration v3 and above allows changing the interval at which metrics are gathered from the cluster. This allows choosing a tradeoff between data resolution and usage. We recommend choosing an interval between 15 and 30 seconds for optimal experience.
+
+In order to change the scrape interval, add the following to your `values-newrelic.yaml`, under the `newrelic-infratructure` section:
+
+```yaml
+common:
+  config:
+    interval: 25s
+```
+
+So it ends up looking like:
+
+```yaml
+global:
+  licenseKey: <var>YOUR_NEW_RELIC_LICENSE_KEY</var>
+  cluster: <var>K8S_CLUSTER_NAME</var>
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      interval: 25s
+```
+
+<Callout variant="important">
+  Setting `interval` to values larger than `45s` is not allowed.
```

Comment:
is discouraged and could be set anyway via config

Reply:
That is up to the user to figure out :P

Reply:
resolved